### PR TITLE
Release bump 1.8.1

### PR DESCRIPTION
**Pull request by raghavaggarwal2308**
#### Fixes
- https://github.com/mattermost/mattermost-plugin-gitlab/pull/419
- https://github.com/mattermost/mattermost-plugin-gitlab/pull/394
- https://github.com/mattermost/mattermost-plugin-gitlab/pull/395
- https://github.com/mattermost/mattermost-plugin-gitlab/pull/429
- https://github.com/mattermost/mattermost-plugin-gitlab/pull/406
- https://github.com/mattermost/mattermost-plugin-gitlab/pull/416

### CODE REVIEW

**Comment by mickmister** [comment]
Note that this branch does not have the changes from https://github.com/mattermost/mattermost-plugin-gitlab/pull/451, so we'll need to manually set the version number in `plugin.json` in this PR

**Review by mickmister — APPROVED** [review]

**Comment by raghavaggarwal2308** [comment]
@mickmister Updated version in plugin.json

**Review by AayushChaudhary0001 — APPROVED** [review]
The above PR has been tested for the following scenarios:-
- Assigned MR's instead of opened MR's
- No notification for commits on MR's
- Notification in case of assignee changed to the new assignee assigned.
- Image support in comment notification
- Ephemeral message to the user in desktop app
- Basic slash commands, notifications, LHS action icons and RHS

The PR was working fine for all the above scenarios, LGTM. Approved.

**Review by hanzei — APPROVED** [review]

File: plugin.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "support_url": "https://github.com/mattermost/mattermost-plugin-gitlab/issues",
     "release_notes_url": "https://github.com/mattermost/mattermost-plugin-gitlab/releases/tag/v1.8.0",
```
Comment (mickmister) [inline plugin.json:7]:
```suggestion
    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-gitlab/releases/tag/v1.8.1",
```

Comment (mickmister) [reply plugin.json:7]: We can change this in marketplace PR

**Review by mickmister — APPROVED** [review]

**Pull request merged**